tests/unit/src/test/scala/tests/DebugProtocolSuite.scala

```diff
@@ -8,9 +8,10 @@ import scala.meta.internal.metals.DebugUnresolvedMainClassParams
 import scala.meta.internal.metals.DebugUnresolvedTestClassParams
 import scala.meta.internal.metals.JsonParser._
 import scala.meta.internal.metals.debug.WorkspaceErrorsException
-import scala.meta.internal.metals.Messages
 
-class DebugProtocolSuite extends BaseLspSuite("debug-protocol") {
+// note(@tgodzik) all test have `System.exit(0)` added to avoid occasional issue due to:
+// https://stackoverflow.com/questions/2225737/error-jdwp-unable-to-get-jni-1-2-environment
```

Comment:
We could add an automatic test during`server.startDebugging` checking if the main class contains the `System.exit(0)` string - it would be relatively fast and straight-forward smoke test preventing recurrence of this issue.

Reply:
Added and it actually detected one bad test.